gpt_engineer/main.py

```diff
@@ -7,10 +7,15 @@
 
 import typer
 
-from gpt_engineer import steps
-from gpt_engineer.ai import AI
-from gpt_engineer.db import DB, DBs
-from gpt_engineer.steps import STEPS
+#from gpt_engineer import steps
+#from gpt_engineer.ai import AI
+#from gpt_engineer.db import DB, DBs
+#from gpt_engineer.steps import STEPS
+
+import steps
+from ai import AI
+from db import DB, DBs
+from steps import STEPS
```

Comment:
This doesn't follow the [styling guide](https://github.com/AntonOsika/gpt-engineer/blob/main/.github/CONTRIBUTING.md) plus relative imports could cause issues, we decided on absolute imports at the beginning and we are sticking to it for now.